frontend/src/pages/home/GroupLeaderHomePage.tsx

```diff
@@ -0,0 +1,73 @@
+import { FunctionComponent, useContext } from 'react'
+import ButtonLink from '../../components/ButtonLink'
+import CheckIcon from '../../components/icons/CheckIcon'
+import CogIcon from '../../components/icons/CogIcon'
+import TruckIcon from '../../components/icons/TruckIcon'
+import InternalLink from '../../components/InternalLink'
+import { UserProfileContext } from '../../components/UserProfileContext'
+import ROUTES, { groupViewRoute } from '../../utils/routes'
+
+const GroupLeaderHomePage: FunctionComponent = () => {
```

Comment:
Group leaders have their own home page! See screenshots in the PR description.

Reply:
awesome!